### ISSUE TITLE
Fishing levelling doesnt match wowhead comments.

### ISSUE THREAD

**Issue by g-valentino** [Wall-core/Everlook-Bugtracker#689]
FIshing leveling seems oddly inconsistent. Either somewhat accurate values, or some incredible absurd values.
By this I mean that either the number of catches is within the expected range, following:
Expected Number of catches  = max (floor(fishing level  / 25) - 2, 1)
Expected as in the averrage of catches for the 25 skill points of the bracket, the further we are from the expected value the most unlikely that is to happen. (If y average is 2, 10 catches for a skill is very unlikely).

Behavior I run into,very consistently while leveling from 134 to 205 (5 hours of continious fishing on level appropiate zone, hillsbrad until 150 and boot bay until 205).

On of multiple instances:
```
Your skill in Fishing has increased to 204.
You receive loot: [Raw Bristle Whisker Catfish].
You receive loot: [Raw Rockscale Cod].
You receive loot: [Raw Rockscale Cod].
You receive loot: [Raw Rockscale Cod].
You receive loot: [Raw Rockscale Cod].
You receive loot: [Raw Rockscale Cod].
You receive loot: [Stranglekelp].
You receive loot: [Raw Spotted Yellowtail].
You receive loot: [Raw Mithril Head Trout].
You receive loot: [Oily Blackmouth].
You receive loot: [Raw Rockscale Cod].
You receive item: [Bright Baubles].
You receive loot: [Raw Mithril Head Trout].
You receive loot: [Oily Blackmouth].
You receive loot: [Raw Rockscale Cod].
You receive loot: [Raw Bristle Whisker Catfish].
You receive loot: [Raw Rockscale Cod].
You receive loot: [Raw Rockscale Cod].
You receive loot: [Oily Blackmouth].
You receive loot: [Stonescale Eel].
You receive loot: [Oily Blackmouth].
You receive loot: [Raw Rockscale Cod].
You receive loot: [Raw Rockscale Cod].
You receive loot: [Raw Spotted Yellowtail].
You receive loot: [Raw Rockscale Cod].
You receive loot: [Raw Rockscale Cod].
You receive loot: [Raw Mithril Head Trout].
You receive loot: [Oily Blackmouth].
You receive loot: [Raw Rockscale Cod].
You receive loot: [Raw Rockscale Cod].
You receive loot: [Raw Spotted Yellowtail].
You receive loot: [Raw Spotted Yellowtail].
You receive loot: [Oily Blackmouth].
You receive loot: [Raw Rockscale Cod].
You receive loot: [Raw Rockscale Cod].
You receive loot: [Raw Mithril Head Trout].
Your skill in Fishing has increased to 205.
```
The average for 200 -> 6 per skillpont. I had a couple 
This is suprising since I levelled fishing in classic and som plenty of times, and never run into this, at least this early. Shiit gets bad past
225 and comments in wowhead share this fact.

Anyway might not be a bug? Might be a just a very bad case? I got a very bad seed on the rng generator? (This was consistent during the entire 6 hours of fishing I did today, while basting the turkey.  Anyway wanted to bring this for closer inspection, seems like some off by one or some odd thing going on, for the behavior to be so inconsistent. (Levelled fishing in nostalrius, elysium, darrowshire, etc and never had this sort of case show so early, and so consisntently).

**Comment by FlagFlayer** [comment]
I think you got unlucky, I levelled fishing to 225 and while I had a few points that took a while, I had a few that were one after another after another (also unlikely)

**Comment by g-valentino** [comment]
Yes you can have points that take a couple of tries. For some contextual information it tooke me 11 hours non stop(that's how long making the turkey took kmn) to level up fishing from 125 to 225 minis some time .

Where most of the time skills would take more than twice the expected amount of time. Distribution wise the chances of independent experiments (samples) going 2* mean is very low, and sometimes all the way up to 4 times the mean.

One of the earliest 146/25 = 6. 6-2 = 4. Then this one almost 40 catches, that is 10 times the mean almos(more like eight 32). Anyway, you can't blame 'unlucky' on a consistent behavior, that statically should al.ost me er happen(n times mean).

It's worth taking a look, specially because it was not even pseudo uniform. It was like if I was with a expected of 4 catches for examples I'll get 2 3 4 5 6 on one hand and on the other 15-40 catches. The gap in these two groups is what makes this behavior very odd.

This is not taking into account that I levelled fishing in classic(like 8 times) and some(once). While after 225 got harder,.leveling up to there was not hard.
Even comments like:
https://us.forums.blizzard.com/en/wow/t/how-long-will-it-take-to-get-fishing-from-1-to-300/400327/9

Where is 10 hrs average for 1-300 I went from 125 to 225, which doesn't even. Include the uptick of higher levels.

So all I'm asking is look into it, don't just wave it off as unlucky, without doing due diligence.

**Comment by FlagFlayer** [comment]
I levelled fishing from 0 to 225 in about 4 hours, which considering 225-300 also takes about 5 hours (from watching fishing videos while gathering loot table data) I think that 10 hour average is correct. What I said is I **think** you got unlucky because you didn't give enough relevant details and followed that by stating "Anyway might not be a bug? Might be a just a very bad case? I got a very bad seed on the rng generator?" While your experience was also completely inconsistent with my own.

11 hours to go from 125 to 225 is absolutely absurd, no sort of RNG can cause that, especially when I can confirm for a fact that it doesn't take that long to level fishing from my own experience on Everlook.

Did you fish in multiple different places in the zone or switch between Inland and Sea fishing? Where exactly in Hillsbrad were you fishing? Did you use baubles? Please give all information that could be relevant so this can get looked into.

**Comment by g-valentino** [comment]
Fished in hillsbrad on the pond just outside Tarren Mill. Did use bait (nightcrawlers i think)
At 146 i went and tried arathi to see, same result. Moved back to the pond outside tarren mill.
At 150i moved to STV, booty bay, just outside the fishing hut(right side of the ramp from thre sea). From this moment on I did not move. Used bait (fromt he fishing vendor in the hut in Booty Bay) all the way to 225.
